src/inmanta/data/__init__.py

```diff
@@ -3000,13 +3000,18 @@ class Parameter(BaseDocument):
     resource_id: m.ResourceIdStr = ""
     updated: Optional[datetime.datetime] = None
     metadata: Optional[JsonType] = None
-    expires: bool = True
+    expires: bool
 
     @classmethod
-    async def get_updated_before_active_env(cls, updated_before: datetime.datetime) -> list["Parameter"]:
+    async def get_updated_before_active_env(
+        cls, updated_before: datetime.datetime, only_expiring: bool = True
```

Comment:
Now I am wondering why you added the `only_expiring` flag here. This query filters based on the `updated` flag. This flag has no meaning when the fact is not expiring, right?